Source/WebCore/dom/Position.cpp

```diff
@@ -142,6 +143,7 @@ Position::Position(Text* textNode, unsigned offset)
     , m_anchorType(PositionIsOffsetInAnchor)
     , m_isLegacyEditingPosition(false)
 {
+    ASSERT(offset >= 0);
```

Comment:
Should I remove this and keep remaining patch?

Reply:
Yes. Removing assertion in other places makes sense because it's literally impossible for m_offset to be negative after  https://github.com/WebKit/WebKit/commit/6bd1d053f70a0dc5dbd3be29f331ee98e647e94b

Reply:
Done.